assets/src/scss/layout/navbar/_navigation-mobile-menu.scss

```diff
@@ -0,0 +1,226 @@
+#navigation-mobile-menu {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  top: 0;
+  inset-inline-start: -100%;
+  height: 100vh;
+  padding: 18px $sp-3;
+  z-index: 9999;
+  overflow-y: auto;
+  background: var(--top-navigation--background, $dark-blue);
+
+  ul {
+    // Reset the default value
+    padding-left: 0;
+    padding-inline-start: 0;
+    list-style: none;
+  }
+
+  & > nav {
+    height: 100%;
+    padding-bottom: $sp-4;
+    overflow-y: scroll;
+  }
+
+ .site-logo {
+    margin-inline-start: 0;
+    line-height: 1;
+
+    img {
+      height: 23px;
+    }
+  }
+
+  .navigation-mobile-menu-header {
```

Comment:
Separator is too close to the logo when there's no language selector

![Screenshot from 2022-03-03 20-58-50](https://user-images.githubusercontent.com/617346/156651011-d3449841-68fe-436c-a3be-fefd1ded0f0c.png)